.cspell.json

```diff
@@ -90,7 +90,13 @@
         "uuidv",
         "vscc",
         "wasm",
-        "Xdai"
+        "Xdai",
```

Comment:
@jagpreetsinghsasan These look like they are already in the file once (duplicates). Could you double check and if they really are then remove them from the diff.

Reply:
Xdai, parameterizable were required, rest weren't.